### DISCUSSION TITLE
TICA data products at MAST

### DISCUSSION THREAD

**Discussion by hmlewis-astro** [lightkurve/lightkurve#1256] [Ideas]
A heads up for the lk team: TICA FFIs are being added to the MAST Portal! These files are currently available for download from MAST via bulk curl scripts, but in the next few weeks, new TICA deliveries will also be available ~weekly from Portal/astroquery. These products will be utilized in the TESScut UI, so there may be some updates needed for lk users to access these files in a way that is similar to how TESS Mission FFIs are accessed.

Documentation on the [TICA webpage](https://archive.stsci.edu/hlsp/tica) will be updated as these changes take place.

**Comment by barentsen** [comment]
Thank you for the heads up @hmlewis-astro!  We'll monitor these changes.  Do you expect the TESScut API may be modified to offer TICA as a cutout option?

**Reply by hmlewis-astro** [reply]
Yes, work to update the TESScut API is ongoing as well. By default, the SPOC Mission data will be used for Sector searches and making cutouts, but we're adding a 'product' keyword that can be changed to select TICA products.

**Reply by barentsen** [reply]
Interesting!  We'll want to update Lightkurve's `search_tesscut` function to support the optional new keyword.
